### ISSUE TITLE
set title in sentence upper

### ISSUE THREAD

**Issue by fredericky123** [willsALMANJ/Zutilo#143]
can we add a shortcut for this?

**Comment by bwiernik** [comment]
Generally, you don't want to do this. You should store your titles in sentence case and then let the citation style handle converting it to title case if needed. Zotero can go from sentence case to title case reliably, but not the other way around.

**Comment by fredericky123** [comment]
> Generally, you don't want to do this. You should store your titles in sentence case and then let the citation style handle converting it to title case if needed. Zotero can go from sentence case to title case reliably, but not the other way around.

but under some circumstances, zotero store the citation in title case automatically, so I need to convert it to sentence case manually.

**Comment by bwiernik** [comment]
Yes, you can right click on any title field and convert the title to sentence case.

**Comment by fredericky123** [comment]
there are so many citations need to be changed, so I would like to a short cut for it.
